src/modules/displayTodo.js

```diff
@@ -0,0 +1,91 @@
+import Icon from '../Icon/more.svg';
+import { getData, addData } from './localStorage.js';
+import populateList from './populateList.js';
+
+// adjusting index
+const adjustIndex = (todoList) => {
+  todoList.forEach((todo, index) => {
+    todo.index = index + 1;
+  });
+};
+
+// removing lists
+const removeTodo = (index, todoList) => {
+  todoList.splice(index, 1);
+  adjustIndex(todoList);
+  // addData(todoList);
+  // populateList(displayTodos(todoList));
+};
+
+// function to display the todo list
+const displayTodos = (todoList) => {
+  const input = document.getElementsByTagName('input')[0];
+  getData(todoList);
+  const listItems = todoList.map((myTodo, index) => {
+    input.value = '';
+
+    // Creating list item
+    const listItem = document.createElement('li');
+    listItem.classList = 'list-item';
+
+    // assigning checkBox for each list
+    const checkBox = document.createElement('input');
+    checkBox.type = 'checkbox';
+    checkBox.className = 'checkbox';
+    checkBox.name = 'checkbox';
+
+    // assigning paragraph for each list
+    const para = document.createElement('input');
+    para.type = 'text';
+    para.className = 'list-para';
+    para.id = `${myTodo.index}`;
+    para.value = `${myTodo.description}`;
+    para.style.border = 'none';
+    para.style.outline = 'none';
+
+    checkBox.addEventListener('change', () => {
+      myTodo.completed = true;
+      if (myTodo.completed === true) {
+        para.style.textDecoration = 'line-through';
+      } else {
+        para.style.textDecoration = 'none';
+      }
+      addData(todoList);
+    });
+
+    // assigning more image container for each list
+    const imgDiv = document.createElement('div');
+    imgDiv.className = 'dot';
+    imgDiv.addEventListener('click', () => {
+      imgDiv.previousElementSibling.style.display = 'block';
+    });
+
+    // assigning more image for each list
+    const imgMore = document.createElement('img');
+    imgMore.className = 'more';
+    imgMore.alt = 'more logo';
+    imgMore.src = `${Icon}`;
+
+    // delete button
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'remove';
+    removeBtn.style.display = 'none';
+    removeBtn.addEventListener('click', () => {
+      removeTodo(index, todoList);
+      adjustIndex(todoList);
+      addData(todoList);
+      populateList(displayTodos(todoList));
+    });
```

Comment:
- [ ] when I edit a task a reload the page it becomes as it was the first time please make sure when editing a task it should be saved in the local storage
<img width="465" alt="Screenshot 2023-02-25 at 23 24 16" src="https://user-images.githubusercontent.com/61790587/221382339-fc63912c-85d2-4078-bb95-8a48a7f4cf95.png">
<img width="667" alt="Screenshot 2023-02-25 at 23 24 22" src="https://user-images.githubusercontent.com/61790587/221382340-99787e5a-12bc-4f9a-b760-3499681f8ba2.png">